.changeset/cuddly-months-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/tag': patch
+---
+
+Enable passing a class name and `data-*` attributes to the `<TagList>` component
```

Comment:
Sorry to be so picky, but I would also not mention `class name` here as it's mostly intended for our internal applications use case and we don't want consumers to use it.
If we don't mention it, we will have more chances that no-one uses it.